src/atoms/atomgraph.jl

```diff
@@ -19,7 +19,7 @@ A type representing an atomic structure as a graph (`gr`).
   graph
 - `laplacian::Matrix{Float32}`: Normalized graph Laplacian matrix, stored to speed up
   convolution operations by avoiding recomputing it every pass.
-- `features::Matrix{Float32}`: Feature matrix of size (# features, # nodes). AtomGraph can
+- `encoded_atom_features::Matrix{Float32}`: Feature matrix of size (# features, # nodes). AtomGraph can
```

Comment:
can we make the type annotation in the docsring match the actual thing? (that seems like something that the formatter should in theory be able to catch; if I someday have infinite time I'll make it do that and PR it, lol)

Reply:
Resolved in d669859.